go/vt/vtctl/workflow/server.go

```diff
@@ -554,6 +557,77 @@ func (s *Server) getWorkflowState(ctx context.Context, targetKeyspace, workflowN
 	return ts, state, nil
 }
 
+// LookupVindexComplete checks if the lookup vindex has been externalized,
+// and if the vindex has an owner, it deletes the workflow.
```

Comment:
I think that the criteria should be the lookup vindex backfill workflow is "done", meaning it was externalized, which in turn means that it's no longer write_only and it has an owner (the source table).

Reply:
Same comment about the `validateExternalized()`